bears/r/FormatRBear.py

```diff
@@ -15,8 +16,38 @@ class FormatRBear:
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-
-        rcode = ("formatR::tidy_source(source =\""+filename+"\"")
+    def create_arguments(filename, file, config_file,
+                         r_comment: bool = True,
+                         r_blank: bool = True,
+                         r_brace: bool = False,
+                         r_arrow: bool = False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                         max_expr_length: int=20):
+        """
+        :param r_comment:       Use to determine whether comments are
+                                kept or not
+        :param r_blank:         Use to determine whether blank lines
+                                are kept or not
+        :param r_brace:         Whether to put the left brace
+        :param r_arrow:         Whether to replace the assign operator = with <-
```

Comment:
we have RST, you can enclose code inside double quotes \`\`
--> enclose `=` and `<-`